extensions/amp-analytics/0.1/vendors/pressboard.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const PRESSBOARD_CONFIG = /** @type {!JsonObject} */ ({
+  'vars': {
+    'mediaId': '',
+    'campaignId': '',
+    'storyRequestId': '',
+    'geoNameId': '',
+    'country': '',
+    'region': '',
+    'city': '',
+    'dbInstance': '',
+    'timeZoneOffset': '',
+  },
+  'requests': {
+    'host': 'https://adserver.pressboard.ca',
+    'common_params': '&amp=1&url=${canonicalUrl}' +
+      '&referrer=${documentReferrer}' +
+      '&rand=${random}' +
+      '&uid=${clientId(_pressboardmedia)}' +
+      '&mid=${mediaId}&cid=${campaignId}&sid=${storyRequestId}' +
+      '&geoid=${geoNameId}&cn=${country}&rg=${region}&ct=${city}' +
+      '&dbi=${dbInstance}&tz=${timeZoneOffset}',
+    'conversion_params': '&asurl=${sourceUrl}' +
+      '&asw=${scrollWidth}&ash=${scrollHeight}' +
+      '&asnw=${screenWidth}&asnh=${screenHeight}' +
+      '&aasnw=${availableScreenWidth}&aasnh=${availableScreenHeight}' +
+      '&asl=${scrollLeft}&ast=${scrollTop}',
+    'conversion': '${host}' +
+      '/track/attention-amp?' +
+      '${common_params}' +
+      '${conversion_params}',
+  },
+  'triggers': {
+    'trackPageview': {
+      'on': 'visible',
+      'request': 'conversion',
+    },
+    'trackAnchorClicks': {
+      'on': 'click',
+      'selector': 'a',
+      'request': 'conversion',
+    },
+    'pageTimer': {
+      'on': 'timer',
```

Comment:
I need to double check on the timer trigger. Once it is checked in here, it will be included as a default trigger. Asking because this can greatly increase the number of requests sent to your server? Is that ok? And you want the timer trigger by default? Thanks

Reply:
can we turn it off based on an input coming back from config?

Reply:
I see. Can I know what the override config looks like? 

Another question I have is:  the timer trigger is the only trigger you have here.  If it is disabled, does that mean there will be no trigger at all?

Reply:
yes, for now all we do is send back user attention variables for our stories, if its not our story, then we don't send back anything

our override for stories that are not ours looks like this: 

{
        vars: {
            mediaId: someMediaId,
        },
        triggers: {
            pageTimer: {
                on: 'never',
            },
        }
    }

Reply:
@zhouyx , any update?

Reply:
@omar84 even with `on: 'never'` a trigger that wait for the custom event `never` will be created. To avoid the waste there, you can use the `enabled` config that we support to make disable the pageTimer trigger. Note `enabled` doesn't support boolean right now, I'll send out a quick PR to fix it. 

Or you can return the `pageTimer` trigger along with the remote config. 

enabled config looks like
```
triggers: {
      pageTimer: {
           enabled: false,
       },
}
```

Reply:
done, I updated the code in our server to return enabled: false instead of on: 'never'

its not live yet, but will be soon.